unlock-protocol-com/blog/base.md

```diff
@@ -0,0 +1,19 @@
+---
+title: Unlock Protocol has been deployed to the Base L2 🔵
+authorName: Julien Genestoux
+publishDate: August 2, 2023
+description: You can use any of the tools available in the Unlock Protocol ecosystem to create membership smart contracts or ticket events on Base.
+image: https://docs.base.org/img/logo.svg
+---
+
+The Unlock Protocol has been deployed to the [Base L2 🔵](https://docs.base.org/)! Our goal at Unlock has always been to support any ecosystem where developers, event organizers, creators, or brands want to deploy **their** membership contracts.
+
+[Coinbase](https://www.coinbase.com/home) (one of our investors) has undeniably contributed massively to the crypto ecosystem, and we believe Base represents a step in democratizing web3 and the onchain web.
+
+As an L2, Base benefits from the “security” budget of Ethereum, while also lowering transaction costs.
+
+## How does it work?
+
+Deploying a membership contract to Base follows the same process as deploying to any other of networks supported by Unlock Protocol. You can deploy your lock(s) using the Unlock Protocol Dashboard (or do it through code) and then use any of the tools available in the Unlock Protocol ecosystem, including using our checkout to sell your NFT memberships, airdrop them, or tap into any of the other capabilities available to the community.
+
+(links to other resources TBD)
```

Comment:
@julien51 what's the ETA for the docs to be live so they can be linked to here?

Reply:
It's live in the docs. https://docs.unlock-protocol.com/core-protocol/unlock/networks 
Do we want to add instructions on how to "add" Base to your wallet + bridge tokens? I can do that.

Reply:
per today's standup, docs are live (networks page on docs)